python/lsst/ts/mtdometrajectory/dome_trajectory.py

```diff
@@ -478,18 +485,25 @@ async def follow_target(self):
         if (
             self.move_dome_elevation_task.done()
             and self.dome_remote.evt_elMotion.has_data
+            and self.enable_el_motion is True
```

Comment:
Since `self.enable_el_motion` is a boolean you could do `and self.enable_el_motion` only no (e.g. remove the unnecessary comparison)?